protocol/communities/manager.go

```diff
@@ -1299,17 +1299,66 @@ func (m *Manager) handleCommunityDescriptionMessageCommon(community *Community,
 	}, nil
 }
 
+func (m *Manager) signEvents(community *Community) error {
```

Comment:
I tried that but `addNewCommunityEvent` does not have access to member's private key. I preferred to delegate signing to manager instead of propagating each `Community` instance with private key.

Reply:
the only cons I see right now is that if somehow you'll get an unsigned event - you will sign it from your name
But ideally this should not happen :)

Reply:
Right, good concern. In current codebase the events are validated when they are received and the ones without signature or with invalid signature are filtered out, so it should not happen.